healthScore/healthScore/views.py

```diff
@@ -27,8 +29,8 @@ def add_mock_data(request):
         hospital.objects.create(name="Hospital D", address="Address D", email="hospital_d@example.com", password="123456", contactInfo="123456781", status="pending")
 
         # Adding hospitalStaff data
-        # hospitalStaff.objects.create(hospitalID=hospital.objects.get(id=1), admin=True, name="Admin A", email="admin_a@hospitala.com", password="pass1234", specialization="", contactInfo="1234567890")
-        # hospitalStaff.objects.create(hospitalID=hospital.objects.get(id=1), admin=False, name="Doctor A", email="doctor_a@hospitala.com", password="pass1234", specialization="Anesthesiology", contactInfo="1234567890")
+        hospitalStaff.objects.create(hospitalID=hospital.objects.get(id=1), admin=True, name="Admin A", email="admin_a@hospitala.com", password="pass1234", specialization="", contactInfo="1234567890")
+        hospitalStaff.objects.create(hospitalID=hospital.objects.get(id=1), admin=False, name="Doctor A", email="doctor_a@hospitala.com", password="pass1234", specialization="Anesthesiology", contactInfo="1234567890")
```

Comment:
Please comment these back so this dummy data is not added in production env

Reply:
OK